lib/vyasa_web/components/control_panel.ex

```diff
@@ -32,18 +33,18 @@ defmodule VyasaWeb.ControlPanel do
             else: "flex flex-col mt-2 space-y-2 hidden"
         }
       >
-        <.button
-          phx-click={JS.push("change_mode", value: %{current_mode: @mode.mode, target_mode: "read"})}
-          class="bg-green-500 text-white px-4 py-2 rounded-md focus:outline-none"
-        >
-          Change to Read
-        </.button>
-        <.button
-          phx-click={JS.push("change_mode", value: %{current_mode: @mode.mode, target_mode: "draft"})}
-          class="bg-red-500 text-white px-4 py-2 rounded-md focus:outline-none"
-        >
-          Change to Draft
-        </.button>
+        <%= for other_mode <- @mode.control_panel_modes do %>
+          <.control_panel_mode_button
```

Comment:
this part is to show what other modes can be shifted to from this mode. This is the flow on how one can shift from one major mode to another